packages/system-next/src/Fiber/context.ts

```diff
@@ -124,6 +124,8 @@ export type FiberRefLocals = Map<FiberRef.Runtime<any>, any>
 
 export const currentFiber = new AtomicReference<FiberContext<any, any> | null>(null)
 
+export const _roots = new Set<FiberContext<any, any>>()
```

Comment:
Should we make this a `LazyValue`? @mikearnaldi @0x706b

Reply:
Couldn't hurt 👍🏻

Reply:
doesn't hurt but also it is a Set so it shouldn't be a big deal (doesn't depend on anything)